sklearn/linear_model/_logistic.py

```diff
@@ -1209,6 +1215,17 @@ def fit(self, X, y, sample_weight=None):
         check_classification_targets(y)
         self.classes_ = np.unique(y)
 
+        if self.multi_class != "deprecated":
+            warnings.warn(
+                (
+                    "'multi_class' was deprecated in version 1.5 and will be removed in"
+                    " 1.7. Use OneVsRestClassifier(LogisticRegression(..)) instead."
```

Comment:
If one specifically sets `multi_class="multinomial"`, then this warning seems out of place.

Reply:
Why is it off? The parameter `multi_class` should best not be used anymore, also not with value `"auto"`.

Reply:
The proposed implementation always triggers the warning as soon as ``multi_class != "deprecated"``. And in addition it gives different instructions depending on the user's intent which is nice. So I think there's nothing wrong here.

Reply:
I don't understand anymore why I thought it was off.